src/librustc_resolve/lib.rs

```diff
@@ -2677,8 +2665,7 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                                    path: &Path,
                                    namespace: Namespace,
                                    check_ribs: bool)
-                                   -> AssocItemResolveResult
-    {
+                                   -> AssocItemResolveResult {
```

Comment:
:+1: breaking before `{` is really good when a `where` clause exists, and further, I think it makes sense to be consistent and just do it everywhere.